epochtalk/epochtalk
# Polls

Added a poll option during thread creation. 

Testing Procedures:

Go to a board and start a new thread.
Create a thread with no poll attached.

Go to a board and start a new thread.
There should be an option to create a poll. Click on this option to show the poll creator.
Try to submit without any input.
Add a Question.
Try to submit with just a question.
Remove one of the two answer options.
Try to submit with just a question.
Add a answer option again and fill out both answers.
Add up to 8 answer and ensure that with each new answer option, the submit button is disabled when there is not text in any answer option.
Submit the thread with the poll.

In the thread:
Try to lock the poll as: an admin, a moderator, as the poll owner.
Ensure that each time it is locked to all other users.
Ensure that voting is not allowed for non-logged in users.
Unlock the poll and vote.
Ensure other users can vote.
Ensure users cannot vote more than once.
